content/tutorials/build-apps/build-a-desktop-wallet-in-javascript.md

```diff
@@ -0,0 +1,1433 @@
+        <button type="button" class="btn btn-primary" id="send-xrp-modal-button">
+            Send XRP
+        </button>
+    </div>
+```
+
+6. In the same file, add said modal dialog:
+```html
+    <div class="modal fade" id="send-xrp-modal" tabindex="-1">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h1 class="modal-title fs-5" id="send-xrp-modal-label">Send XRP</h1>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div class="modal-body">
+                    <div class="input-group mb-3">
+                        <input type="text" class="form-control" placeholder="r9jEyy3nrB8D7uRc5w2k3tizKQ1q8cpeHU" id="input-destination-address">
+                        <span class="input-group-text">To (Address)</span>
+                    </div>
+                    <div class="input-group mb-3">
+                        <input type="text" class="form-control" placeholder="12345" id="input-destination-tag">
+                        <span class="input-group-text">Destination Tag</span>
+                    </div>
+                    <div class="input-group mb-3">
+                        <input type="text" class="form-control" placeholder="100" id="input-xrp-amount">
+                        <span class="input-group-text">Amount of XRP</span>
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Cancel</button>
+                    <button type="button" class="btn btn-primary" id="send-xrp-submit-button">Send</button>
+                </div>
+            </div>
+        </div>
+    </div>
+```
+
+7. Update `view/renderer.js` by adding:
+
+```javascript
+const modalButton = document.getElementById('send-xrp-modal-button')
+const modalDialog = new bootstrap.Modal(document.getElementById('send-xrp-modal'))
+modalButton.addEventListener('click', () => {
+    modalDialog.show()
+})
+
+const destinationAddressEl = document.getElementById('input-destination-address')
+const destinationTagEl = document.getElementById('input-destination-tag')
+const amountEl = document.getElementById('input-xrp-amount')
+const sendXrpButtonEl = document.getElementById('send-xrp-submit-button')
+
+sendXrpButtonEl.addEventListener('click', () => {
+    modalDialog.hide()
+    const destinationAddress = destinationAddressEl.value
+    const destinationTag = destinationTagEl.value
+    const amount = amountEl.value
+
+    window.electronAPI.onClickSendXrp({destinationAddress, destinationTag, amount})
+})
+
+window.electronAPI.onSendXrpTransactionFinish((_event) => {
+    destinationAddressEl.value = ''
+    destinationTagEl.value = ''
+    amountEl.value = ''
+})
+```
+
+Now, Run the following command:
+
+```console
+npm run start
+```
+
```

Comment:
Just for testing the app out, I think it'd be helpful if someone were to just click "Send XRP" then "Send" without changing any of the defaults, it should actually use those defaults. (So sending to the "faded text address" 100 XRP. Having to go find your own address seems like an extra step, so if we want them to do that we should explain how to find a testnet address.

Reply:
Done, switched the placeholders to prefilled values and added a section on how to get a new account address. Also added an alert that indicates the transaction result so that the user has at least some feedback.

Reply:
Awesome :)